folding-schemes-solidity/templates/groth16_verifier.askama.sol

```diff
@@ -0,0 +1,169 @@
+{{ sdpx }}
+/*
+    Copyright 2021 0KIMS association.
+
+    This file is generated with [snarkJS](https://github.com/iden3/snarkjs).
```

Comment:
`This file is generated with snarkJS` --> is it? or is a fork of the template from snarkjs?
Maybe we it can keep the 0KIMS/iden3 license but updating the sentence? Maybe adding a link to the original template from snarkjs repo?
(I have not much idea on the licensing topic :S)

Reply:
I'm not too familiar with gpl-3 but I think that is ok? We should still figure out which and how the license should be included in general.
So I think that's a fair point and added this to the template with [8c843e5](https://github.com/privacy-scaling-explorations/folding-schemes/pull/70/commits/8c843e5a46ce4f7f739473f898b9d248390e3dc5)